docker-for-windows/index.md

```diff
@@ -13,23 +13,26 @@ redirect_from:
 title: Get started with Docker for Windows
 ---
 
-Welcome to Docker for Windows! Docker is a full development platform for
-creating containerized apps, and Docker for Windows is the best way to get
-started with Docker _on Windows_.
+Welcome to Docker for Windows!
+
+Docker is a full development platform for creating containerized apps, and
+Docker for Windows is the best way to get started with Docker _on Windows_.
 
 > See [Install Docker for Windows](install.md){: target="_blank" class="_"} for information on system requirements and stable & edge channels.
 
 ## Test your installation
 
-1.  Open "Command Prompt (`cmd`) and run `docker --version ` to ensure that you have a supported version of Docker:
+1.  Open a terminal window (Command Prompt or PowerShell, _but not_ PowerShell ISE).
+
+2.  Run `docker --version ` to ensure that you have a supported version of Docker:
```

Comment:
spurious whitespace

Reply:
If you mean the 2 spaces after the number, not spurious. It's 2 "atom" tabs which makes indenting code easier.

Reply:
I mean "`docker --version `".